docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -81,22 +81,33 @@ OpenVM supports generating three types of proofs. The sections below describe ho
 
 ### Generating App Proofs
 
-After building and transpiling a program, you can then generate a proof. To do so, you need to commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+After building and transpiling a program, you can then generate a proof. To do so, you need to convert your ELF into a `VmExe`, commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+
+The SDK, however, combines the first few steps for you. Create a prover and pass in your input as a `StdIn` to generate an app proof.
 
 ```rust
-// [!include ~/snippets/examples_sdk/sdk_stark.rs:proof_generation]
+// [!include ~/snippets/examples_sdk/sdk_app.rs:proof_generation]
 ```
 
 For large guest programs, the program will be proved in multiple continuation segments and the returned `proof: ContinuationVmProof` object consists of multiple STARK proofs, one for each segment.
 
 ### Verifying App Proofs
 
-After generating a proof, you can verify it. To do so, you need your verifying key (which you can get from your `AppProvingKey`) and the output of your `generate_app_proof` call.
+After generating a proof, you can verify it. To do so, you need your verifying key (which you can get from your `AppProvingKey`) and the output of your `prove` call.
```

Comment:
I prefer now that we say you get vkey from `sdk.app_keygen()` since it allows you to store the vkey separately from the proving key.